readme.md

```diff
@@ -0,0 +1 @@
+Application that scrapes title, image, genre, director and rate of the movie form the given number of pages on Filmweb.
```

Comment:
Now we gotta go better, but let's create a new task for it.
https://github.com/xmateq/filmweb-scrape/issues/5